tfjs-backend-webgpu/src/flags_webgpu.ts

```diff
@@ -89,3 +89,6 @@ ENV.registerFlag('WEBGPU_CONV_SEPARATE_IM2COL_SHADER', () => false);
  * etc.). 'unary,conv2d' to print both unary and conv2d.
  */
 ENV.registerFlag('WEBGPU_PRINT_SHADER', () => '');
+
+/** Experimental flag, whether enter compile only phase. */
+ENV.registerFlag('ENGINE_COMPILE_ONLY', () => false);
```

Comment:
In webgl backend, 'ENGINE_COMPILE_ONLY' has been used. We could either move 'ENGINE_COMPILE_ONLY' from webgl to core, or rename the flag here to 'WEBGPU_*'. 

We would unify the feature for webgl and webgpu eventually. Right now, since the feature usage are different between webgl and webgpu, (for example, webgpu does not need to call `getUniformLocations()`), we could have different names for two backends to avoid interfere.

What do you think? @qjia7 @axinging Feel free to say no, and that's just my thoughts.

Reply:
+1, @axinging Please change the flag name to `WEBGPU_ENGINE_COMPILE_ONLY`. Thanks.